docs/dev/dev-mode.md

```diff
@@ -13,70 +13,103 @@ Description: Developer mode (dev mode) is intended to be used for testing and de
-- The Kubernetes node must have hardware virtualization support.
-  - To validate the support, use this command:
-
-    ```bash
-    cat /proc/cpuinfo | grep vmx
-    ```
-- If you are using an [RKE](https://rancher.com/docs/rke/latest/en/) cluster, `ipv4.ip_forward` must be enabled for the CNI plugin in order for pod network to work as expected.
 
 ## Installation
 
-Harvester can be installed on a Kubernetes cluster in the following ways:
-
-- Using the [Helm](https://helm.sh/) CLI.
-- As a Rancher catalog app, in which case the [harvester/harvester](https://github.com/harvester/harvester) repo is added to the Rancher Catalog as a Helm `v3` app
+Harvester can be installed on a Kubernetes cluster by using the [Helm](https://helm.sh/) CLI.
 
 Please refer to the Harvester [Helm chart](https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/README.md) for more details on installing and configuring the Helm chart.
 
-### Option 1: Install using Helm
+1. Create the cattle-system namespace
+   ```bash0
+   kubectl create ns cattle-system
+   ```
+
+1. Add the rancher-latest helm repo
+   ```bash
+   helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+   ```
+
+1. Install Rancher chart
+   ```bash
+   helm install rancher rancher-latest/rancher \
+   --namespace cattle-system \
+   --set tls=external \
+   --set rancherImagePullPolicy=IfNotPresent \
+   --set rancherImage=rancher/rancher \
+   --set rancherImageTag=v2.6.3-harvester1 \
+   --set noDefaultAdmin=false \
+   --set features="multi-cluster-management=false\,multi-cluster-management-agent=false" \
+   --set useBundledSystemChart=true \
+   --set bootstrapPassword=admin
+   ```
+
+1. Change the 'status.provider' of the local cluster object to "harvester"
+   ```bash
+   kubectl edit clusters.management.cattle.io local
+   ```
 
-1. Clone the GitHub repository:
+1. Clone the GitHub repository
    ```bash
    git clone https://github.com/harvester/harvester.git --depth=1
    ```
 
-1. Go to the Helm chart:
+1. Go to the Helm chart
    ```bash
    cd harvester/deploy/charts
    ```
 
-1. Install the Harvester chart with the following commands:
+1. Create the harvester-system namespace
    ```bash
-   ### To install the chart with the release name `harvester`:
-
-   ## Create the target namespace
    kubectl create ns harvester-system
+   ```
+
+1. Install the Harvester crd chart
+   ```bash
+   helm install harvester-crd ./harvester-crd --namespace harvester-system
+   ```
 
-   ## Install the chart to the target namespace
-   helm install harvester harvester \
-   --namespace harvester-system \
-   --set longhorn.enabled=true,minio.persistence.storageClass=longhorn
+1. Install the Harvester chart
+   ```bash
+   ## in order to use the service type LoadBalancer and create a vip in control-plane nodes, we need to enable kubevip
```

Comment:
```suggestion
   ## In order to use the service type LoadBalancer and create a vip in control-plane nodes, we need to enable kubevip.
```

Reply:
Just a suggestion to make the comments following the "##" start with capitals and end with punctuation for consistency.